front/src/applications/operationalStudies/views/v2/ManageTrainScheduleV2.tsx

```diff
@@ -18,6 +18,7 @@ import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
 import { RollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector';
 import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
 import TimesStops from 'modules/timesStops/TimesStops';
+import { TableType } from 'modules/timesStops/types';
```

Comment:
add type prefix

Reply:
it’s an enum so I need to use it at runtime

Reply:
I reviewed all the imports, I think the other spots were also for this enum, I think the rest of imports are ok